### ISSUE TITLE
Symbol processing stops processing new packages

### ISSUE THREAD

**Issue by pvandervelde** [pvandervelde/nAnicitus#1]
After an amount of time the symbol processor stops processing new symbol packages. The log files indicate that the new packages are detected and that a new processing Task is started but the packages are never processed.

**Comment by pvandervelde** [comment]
It looks like this problem is caused by an file access error (i.e. the file is still being written to probably). The following log segment seems to point to the problem:

```
Error: Loading of package failed. Error was: System.IO.IOException: The process cannot access the
    file 'C:\integration\nanicitus\uploads\Rakon.EngineeringTools.2.6.3.4.symbols.nupkg' because it is being 
    used by another process.
    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
    at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, 
        Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, 
        SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, 
        Boolean checkHost)
    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
    at NuGet.ZipPackage.<>c__DisplayClass2.<.ctor>b__0()
    at NuGet.ZipPackage.EnsureManifest()
    at NuGet.ZipPackage..ctor(String filePath, Boolean enableCaching)
    at Nanicitus.Core.SymbolIndexer.LoadSymbolPackage(String packageFile).
```

**Comment by pvandervelde** [comment]
One solution to the problem could be to detect the file lock and then just wait a bit before processing the package. We'd expect the file locks to be lifted pretty quickly. We could for instance try to load the same package 3 times, each time waiting 1 second between tries. 

We should probably limit our tries to known conditions only (e.g. file lock) to prevent packages from permanently locking up the symbol processing.

**Comment by pvandervelde** [comment]
Fixed in 0.1.4.0
